### ISSUE TITLE
Energy Legend: make all text black and add color swatches

### ISSUE THREAD

**Issue by arouinfar** [phetsims/pendulum-lab#217]
In https://github.com/phetsims/griddle/issues/49, `labelColor` was added to bar graphs, and the is default black. The primary motivation for this change was the contrast ratio between the colored text and the white background was poor (see values in https://github.com/phetsims/griddle/issues/49#issuecomment-622483092).

Another place that the colored text appears is in the Energy Legend:
![image](https://user-images.githubusercontent.com/8419308/80826498-94c1a100-8b9f-11ea-899a-9635b230a75e.png)

All text should be made black, but to compensate, we should add a color swatch next to each entry. This mockup comes from masses-and-springs, but the idea should be clear.
![image](https://user-images.githubusercontent.com/8419308/80825415-90948400-8b9d-11ea-9c1e-314bcce6a7de.png)
